Sources/SwiftGRPC/Runtime/ServiceClient.swift

```diff
@@ -55,6 +55,13 @@ open class ServiceClientBase: ServiceClient {
     metadata = Metadata()
   }
 
+  /// Create a client using a pre-defined channel.
+  public init(channel: Channel) {
+    gRPC.initialize()
```

Comment:
I think we can remove this `gRPC.initialize()` call, given that this is now done by `Channel.init`. Also in the other constructors above and below.

Also, I'd suggest moving this constructor below or above all the other ones?